tox.ini

```diff
@@ -0,0 +1,207 @@
+[tox]
+minversion = 3.14.0
+#requires=tox-conda
+ignore_basepython_conflict = true
+# these are the environments that will run when you
+# execute `tox` in the command-line
+# bellow you will find explanations for all environments
+envlist =
+    build
+    docs
+    py37
+    py38
+    py39
+
+# configures which environments run with each python version
+[testenv]
+basepython =
+    {py37,docs}: {env:TOXPYTHON:python3.7}
+    {py38}: {env:TOXPYTHON:python3.8}
+    {py39}: {env:TOXPYTHON:python3.9}
+    {build,prreqs,lint,radon,safety}: {env:TOXPYTHON:python3}
+passenv = *
+#deps = -r{toxinidir}/cirequirements.txt
+
+# configures the unittest environment for python 3.6
+[testenv:py37]
+setenv =
+    PYTHONPATH={toxinidir}/tests
+    PYTHONUNBUFFERED=yes
+user_develop = false
+# installs dependencies we need for testing
+# by using tox the developer don't need to manage this dependencies
+# him/herself
+conda_deps =
+    mdanalysis>=1
```

Comment:
@orbeckst @PicoCentauri @IAlibay 
note here I am putting MDA 1 just for the sake of having this passed. ASAP MDAv2 is released to conda we can update this.

Reply:
Just an update on this, we'll release more on the MDA issue tracker in a little while.

Because of the amount of changes that are happening, MDA is going to do a beta release via pypi first on the 10th. We won't be releasing on conda until the 28th of May (i.e. the last day of the MDAnalysis workshop).

Reply:
I believe that is not a problem at all. MDA conda installation here is only activated for tests. To develop mdacli we can have MDA2 installed locally (manually), as long we don't ask tox to install MDA2 from conda forge everything is okay. We will update the CI at the end of the month.